src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConditionalOperatorTests.cs

```diff
@@ -13,6 +13,532 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+            verifier.VerifyDiagnostics();
+            verifier.VerifyMethodBody("C.Compare", """
+                {
+                  // Code size       27 (0x1b)
+                  .maxstack  2
+                  .locals init (int V_0, //tmp1
+                                int V_1, //tmp2
+                                int V_2)
+                  // sequence point: {
+                  IL_0000:  nop
+                  // sequence point: int tmp1 = (x > y) ? 1 : 0;
+                  IL_0001:  ldarg.0
+                  IL_0002:  ldarg.1
+                  IL_0003:  bgt.s      IL_0008
+                  IL_0005:  ldc.i4.0
+                  IL_0006:  br.s       IL_0009
+                  IL_0008:  ldc.i4.1
+                  IL_0009:  stloc.0
+                  // sequence point: int tmp2 = (x < y) ? 1 : 0;
+                  IL_000a:  ldarg.0
+                  IL_000b:  ldarg.1
+                  IL_000c:  blt.s      IL_0011
+                  IL_000e:  ldc.i4.0
+                  IL_000f:  br.s       IL_0012
+                  IL_0011:  ldc.i4.1
+                  IL_0012:  stloc.1
+                  // sequence point: return tmp1 - tmp2;
+                  IL_0013:  ldloc.0
+                  IL_0014:  ldloc.1
+                  IL_0015:  sub
+                  IL_0016:  stloc.2
+                  IL_0017:  br.s       IL_0019
+                  // sequence point: }
+                  IL_0019:  ldloc.2
+                  IL_001a:  ret
+                }
+                """);
+        }
+
+        [Fact, WorkItem(61483, "https://github.com/dotnet/roslyn/issues/61483")]
+        public void Branchless_Operations()
+        {
+            var source = """
+                using static System.Console;
+
+                class C
+                {
+                    static void Main() => M(1, 0, true);
+                    static void M(int x, int y, bool b)
+                    {
+                        Write(x == y ? 1 : 0);
+                        Write(x == y ? 0 : 1);
+                        Write(x < y ? 1 : 0);
+                        Write(x < y ? 0 : 1);
+                        Write(x > y ? 1 : 0);
+                        Write(x > y ? 0 : 1);
+                        Write(b ? 0 : 1);
+                        Write(!b ? 0 : 1);
+                        Write(x <= y ? true : false);
+                        Write(x <= y ? false : true);
+                        Write(x != y ? (byte)1 : (byte)0);
+                        Write(x != y ? 1 : (sbyte)0);
+                        Write(x != y ? (short)1 : (short)0);
+                        Write(x != y ? (ushort)1 : 0);
+                        Write(x != y ? 1U : 0);
+                        Write(x != y ? 1L : 0);
+                        Write(x != y ? 1UL : 0);
+                        Write(x != y ? (nint)1 : 0);
+                        Write(x != y ? 1 : (nuint)0);
+                        Write(x < y ? (char)0 : (char)1);
```

Comment:
>(char)0 : (char)1 [](http://example.com/codeflow?start=38&length=17)

Since this hits a unique code path, I think we should also test the ```1 : 0``` case.